conditionals.md

```diff
@@ -0,0 +1,227 @@
+## Conditionals Statements
+
+Conditional statements is like a decision we make based on circumstances. For example if i wake up very early, i do some exercise e.g Road work and cardio, else if i wake up 30mins late, i skip road work, and do just cardio, else i eat breakfast and write some code. These if-else decisions can be modeled in code by creating conditional statements. A conditional statement checks specific condition(s) and performs a task based on the condition(s).
+
+### `if` Statement
+
+In programming, task can be perform based on a condition using an `if` statement. example:
+
+```js
+if (true) {
+    console.log('Print this message!');
+} // output: Print this message!
+```
+The if statement is composed by writing:
+
+- The `if` keyword followed by a set of parentheses () which is followed by a code block, or block statement, indicated by a set of curly braces {}.
+- Inside the parentheses (), a condition is provided that evaluates to true or false.
+- If the condition evaluates to true, the code inside the curly braces {} runs, or executes.
+- If the condition evaluates to false, the block won’t execute.
+
+### `else` Statement
+
+If we wanted to add some default behavior to the `if` statement, we can add an `else` statement to run block of code, when the condition evaluate to false. example:
+
+```js
+if (false) {
+    console.log('The code in this block will not run');
+} else {
+    console.log('But the code in this block will');
+} // output: But the code in this block will
+```
+
+An `else` statement must be paired with an `if` statement, and together they are referred to as an `if...else` statement.
+
+ - `else` keyword is used following the code block of an `if` statement.
+- Has a code block that is wrapped by a set of curly braces {}.
+- The code inside the `else` statement code block will execute when the `if` statement’s condition evaluates to false.
+`if...else` statements allow us to automate solutions to yes-or-no questions, also known as binary decisions.
+
+### Comparison Operator
+
+When writing conditional statement, we sometimes need to use different types of operator to compare values. these operators are called comparison operators.
+#### List of some comparison operators and their syntax
+
+- less than `<`
+- greater than `>`
+- less than or equal to `<=`
+- greater than or equal to `>=`
+- equal to `===`
+- not equal to `!==`
```

Comment:
There's also `==` and `!=`, I would recommend investigating this and writing about the difference.

Reply:
I wrote something about the operators a little but it's explained it.

Reply:
I don't understand what you meant here. :thinking: